.kuzzlerc

```diff
@@ -7,6 +7,12 @@
   */
   "serviceSettingsCollection": "services",
 
+  "default": {
```

Comment:
I agree, this is not clean.
I am sorry but on my side, I don't like having some parameters at the root of the rc file.

What about restructuring things a bit?:

```json
{
  "core": {
    "server": {
      "rest": {
        "port": 7511
      }
    },
    "requests": {
      "maxConcurrentRequests": 50,
      "maxRetainedRequests": 200
    }
  },
  "services": {
    "common": {
      "collection": "services",
      "initTimeout": 10000,
      "retryInterval": 1000
    },
    "service1": {
      "host": "api",
      "port": 7331
    }
  },
  "plugins": {
    "common": {
      "pipeWarnTime": 40,
      "pipeTimeout": 250,
      "dataCollection": "plugins"
    },
    "kuzzle-plugin-logger": {},
    "kuzzle-plugin-auth-passport-local": {}
  }
}
```